wiki/Help_Center/en.md

```diff
@@ -136,95 +136,239 @@ A couple of hours at the absolute maximum will clear the status for most people
 
-Make sure you're not running osu! in compatibility mode through the osu!.exe propreties and make sure that you have a default browser set.
+**Make sure you're not running osu! in Windows compatibility mode, and that you also have a default web browser defined in your system settings.**
 
-Right click the osu!.exe file in the osu! installation folder and click on Propreties, then go to the Compatibility tab and make sure that the checkbox under Compatibility mode is **not** checked.
+This issue is sometimes caused by an interaction with compatibility mode, and can also be caused by the game simply not finding a default browser to open the web page link with.
 
-If you're not running osu! in compatibility mode and the issue persists, make sure you have a default browser set. For some users that have Chrome as their default browser the issue will still occur for unknown reasons, so try setting Internet Explorer as the default browser and see if your issue persists after that.
+#### Is your game running under Windows compatibility mode?
+
+To check or change if your game is running in compatibility mode under Windows, try the following steps:
+
+1. Open your osu! installation folder and look for `osu!.exe`.
+2. Right click on `osu!.exe` and select `Properties` from the dropdown menu.
+3. Navigate to the `Compatibility` tab.
+4. Look for a checkbox titled `Run this program in compatibility mode` within a section titled `Compatibility mode`.
+5. If this checkbox is ticked, click it once more to disable the functionality.
+6. If your game is already open, close the client and reopen the game once more.
+
+#### Do you have a default web browser set?
+
+**Under the vast majority of all circumstances, most Windows installations will have a default web browser set.**
+
+To check, try the following steps (under Windows 8 and above):
+
+1. Open the Start menu.
+2. Type `default` into the search bar within the Start menu.
+3. If you see a menu entry titled `Default app settings`, click it. If you don't, expand your search to include `default app settings`.
+4. Scroll down to the `Web browser` heading and check to make sure that a browser which you have installed is properly set.
 
 ### I can't login or connect to the game at all!
 
-Please ensure that osu! is allowed full access through whichever firewall or anti-virus software you are using - internet connectivity is required to log into our game servers.
+**Ensure that osu! can properly access the internet through any firewall or anti-virus suites you have installed on your computer.**
+
+Here are a few ways of allowing applications to have internet access through several common anti-virus suites:
+
+#### Allowing a program through Windows Firewall
+
+**On most Windows installations, Windows Firewall will be the default firewall software.**
+
+To make sure osu! can access the internet, we'll need to allow it to communicate through Windows Firewall. Here's how to check and add osu! to the program's list:
```

Comment:
`To make sure *osu!* can access the internet, we will need to allow it to communicate through Windoes Firewall. Here is a guide on how to check and add *osu!* to the program list`